packages/relay-sdk/src/index.ts

```diff
@@ -0,0 +1,132 @@
+
+import { RELAY_URL_HABANERO, RELAY_URL_MANZANO } from '@lit-protocol/constants';
+import { assertNotNull, assertStatus, assertString, getProp } from './util';
+
+type SupportedNetworks = 'habanero' | 'manzano';
```

Comment:
we should use LIT_NETWORKS for this instead of our own string values

Reply:
Is there some way to filter the networks by supported features? I looked into this, but I would have ended up needing to hard code a union anyways since I need to filter out the other networks.